src/main/groovy/com/bmuschko/gradle/docker/DockerJavaApplicationPlugin.groovy

```diff
@@ -58,7 +58,15 @@ class DockerJavaApplicationPlugin implements Plugin<Project> {
                     compositeExecInstruction.entryPoint(project.provider(new Callable<List<String>>() {
                         @Override
                         List<String> call() throws Exception {
-                            ["java", "-cp", "/app/resources:/app/classes:/app/libs/*", getApplicationPluginMainClassName(project)]
+                            List<String> entrypoint = ["java"]
+                            List<String> jvmArgs = dockerJavaApplication.jvmArgs.get()
+
+                            if (!jvmArgs.empty) {
+                                entrypoint.addAll(jvmArgs)
+                            }
+
+                            entrypoint.addAll(["-cp", "/app/resources:/app/classes:/app/libs/*", getApplicationPluginMainClassName(project)])
+                            entrypoint
```

Comment:
I'm curious, as a Groovy newbie, why not reduce these lines to
```groovy
[
    "java",
    *dockerJavaApplication.jvmArgs.get(),
    "-cp", "/app/resources:/app/classes:/app/libs/*",
    getApplicationPluginMainClassName(project)
]
```
?

Reply:
Achieves the same thing with less code. I noticed that developers with little Groovy and/or Gradle experience don't understand the meaning of DSL or language-specific symbolism. The spread operator is great to make things more concise but confuses users that don't know what it means (and the implication of handling empty collections). Feel free to submit a PR if you feel strongly about it.

Reply:
I was mostly curious since I've recently started using Groovy and haven't yet developed an intuition for these things. So I very much appreciate that you took the time to describe your reasoning.

In any case I think the existing code is just fine and I suspect it and the example I gave above will compile to basically the same thing anyway.

Thanks a lot for developing this plugin, it's been a great help!